pkg/bootstrap/bootstrapkubeconfig.go

```diff
@@ -39,19 +39,27 @@ import (
 )
 
 const (
-	bootstrapSASuffix = "bootstrap-sa"
+	bootstrapSASuffix           = "bootstrap-sa"
+	apiServerInternalEndpoint   = "https://kubernetes.default.svc:443"
```

Comment:
I am thinking if we deploy the MCE on a ROSA-HCP(hosted control plane), this seems not correct, this is the hosting cluster API server address, not the hcp API server address, right?
we need to test this case before merging.